charts/ocis/templates/storageusers/deployment.yaml

```diff
@@ -75,6 +75,14 @@ spec:
             - name: STORAGE_USERS_DATA_SERVER_URL
               value: "http://{{ .appName }}:9158/data"
 
+            # propagation strategy
+            {{- if .Values.services.storageusers.asyncPropagation.enabled }}
+            - name: OCIS_DECOMPOSEDFS_PROPAGATOR
+              value: "async"
```

Comment:
owncloud.dev says:

> The propagator used for decomposedfs. At the moment, only ‘sync’ is fully supported, ‘async’ is available as an experimental option.

(from https://owncloud.dev/services/storage-users/)

Reply:
> It was never productized this because we never had the time to test this in kubernetes. We did that yesterday and it allowed all 1000 files to be uploaded without error.

Understood. Can we change the value for the oCIS product / oCIS Chart to hardcoded async in this case now? I'd like to keep the number of configurations permutations low. Same story as the async postprocessing back then...

Reply:
@micbar do you have an opinion on this?